dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -452,13 +459,34 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 								}
 							/>
 						</Island>
+						{isMainMediaTest && (
+							<Hide from="desktop">
```

Comment:
This approach of hiding the main media means this html will exist twice on the page. 

This has impacts in terms of page weight and could be a risk in terms of other code looking for these elements (and not expecting to find more than one).

Could we look to use the [css order property](https://developer.mozilla.org/en-US/docs/Web/CSS/order) here instead?

Reply:
Thanks for this, I can see this  and definitely happy to chat about this further.

I have taken a look at css order and, from what I understand, I don't think it's suitable here for the following:
Firstly, if i understand the property correctly, the elements need to be within the same contain which wouldn't be the case here; below desktop is nested within the headline grid whilst above desktop is nested within the live grid. 
Secondly, as @JamieB-gu mentioned, order is visual only, and will not change the order of content for assistive technology. This is a [known concern ](https://developer.mozilla.org/en-US/docs/Web/CSS/order#accessibility_concerns) for this property and means we'll have a different experience for users using assistive technology